conventional_precommit_linter/hook.py

```diff
@@ -142,6 +154,11 @@ def print_report(commit_type: str, commit_scope: Optional[str], commit_summary:
             f"{_get_icon_for_rule(rules_output_status['error_summary_capitalization'])} {_color_orange('<summary>')} must start with an uppercase letter"
         )
 
+    if not args.allow_breaking:
```

Comment:
@kumekay Could you please rebase and move it under the line `{_color_purple('<type>')} is mandatory, use one of the following ...` to keep the type errors together?

Also small style note: `type` should be colored purple. You might consider rewording it this way to be consistent with the rest of the messages:
```python
    if not args.allow_breaking:
        rule_messages.append(
            f"{_get_icon_for_rule(rules_output_status['error_breaking'])} {_color_purple('<type>')} must not include {_color_purple('!')} to indicate a breaking change"
        )
```

<img src="https://github.com/espressif/conventional-precommit-linter/assets/33937168/4042d66a-6e8e-4b70-a4e8-3d0fd4e0a827" width=500px>